src/move/data/dataloaders.py

```diff
@@ -10,9 +10,23 @@
 from move.core.typing import BoolArray, FloatArray
 
 
-class MOVEDataset(TensorDataset):
-    "Characterizes a dataset for PyTorch"
+class MOVEDataset(TensorDataset):    
+    """
+    Characterizes a dataset for PyTorch
+
+    Attributes:
+        TensorDataset (class): class to represent data as list of tensors
+        
+    Args:
+        cat_all (torch.Tensor, optional): categorical input matrix (N_patients, N_variables x N_max-classes. Defaults to None.
```

Comment:
Instead of `N_patients` use `num_samples` (the datasets do not necessarily describe patients).
Replace `N_variables` with `num_features` and `N_max_classes` with `num_categories`.

Reply:
Fixed it: 3ad2d0c